content/developer/reference/backend/views.rst

```diff
@@ -1014,6 +1014,124 @@ Generic structure
 
 .. todo:: widgets?
 
+Settings Form View
+------------------
```

Comment:
Shouldn't this section be a sub-section of Form?
```suggestion
~~~~~~~~~~~~~~~~~~
```
The title could be **Settings components** to be consistent with the other sub-sections, for example.

Reply:
The settings view, is not a component of the form view. It's a jsClass, a customization of the form view, that have a search bar (form view don't have one), a sidebar, and had its own components. It's for these reasons that we put-it as an independent view.